tests/testthat/test-backend-oracle.R

```diff
@@ -20,3 +20,14 @@ test_that("queries translate correctly", {
     sql("^SELECT [*] FROM [(]SELECT [*]\nFROM [(]`df`[)] [)] `[^`]*` WHERE ROWNUM [<][=] 6")
   )
 })
+
+test_that("vectorised translations", {
```

Comment:
This feels like slightly too much testing because you're effectively testing `sql_paste_infix()`, which should be tested elsewhere. I think a single "sanity" check for `paste()` and `paste0()` should be adequate.